### PR TITLE
Sb/add new notes merge for v1.2 launch

### PR DESCRIPTION
**Pull request by grenerad**
V1.2 deployment; merging all new adds and edits from sb/add-new-notes into main; Hard Times and Little Dorrit pages new, most other pages contain small edits or reworked text; footer and copyright info updated

### CODE REVIEW

File: src/pages/notes/hard-times/mirador.astro [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+import MiradorFullScreenLayout from "../../../layouts/mirador-full-screen-layout.astro";
+import Mirador from "../../../components/Mirador";
+
+const manifest =
+  "https://dickensnotes.github.io/dickens-annotations/img/derivatives/iiif/HardTimesTranscription/manifest.json";
+---
+
+<MiradorFullScreenLayout title="Hard Times Mirador viewer" description="TODO">
```
Comment (csbailey5t) [inline src/pages/notes/hard-times/mirador.astro:9]: Need to fill in this TODO in the meta description

File: src/pages/notes/little-dorrit/mirador.astro [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+import MiradorFullScreenLayout from "../../../layouts/mirador-full-screen-layout.astro";
+import Mirador from "../../../components/Mirador";
+
+const manifest =
+  "https://dickensnotes.github.io/dickens-annotations/img/derivatives/iiif/littledorrittranscription/manifest.json";
+---
+
+<MiradorFullScreenLayout title="Little Dorrit Mirador viewer" description="TODO">
```
Comment (csbailey5t) [inline src/pages/notes/little-dorrit/mirador.astro:9]: Need to fill in this TODO in the meta description

**Comment by csbailey5t** [comment]
@grenerad a couple of small things to finish out, but this looks great.

File: src/pages/notes/hard-times/mirador.astro (hunk 1 above)
Comment (grenerad) [reply src/pages/notes/hard-times/mirador.astro]: added description in metadata

File: src/pages/notes/little-dorrit/mirador.astro (hunk 2 above)
Comment (grenerad) [reply src/pages/notes/little-dorrit/mirador.astro]: added description in metadata

**Pull request merged**